aws-datastore/src/test/java/com/amplifyframework/datastore/appsync/AppSyncMocking.java

```diff
@@ -248,5 +249,25 @@ public SyncConfigurator mockSuccessResponses() {
             );
             return SyncConfigurator.this;
         }
+
+        /**
+         * Triggers an exception when invoking the sync method.
+         * @param <T> Type of models for which a response is mocked
+         * @return The same Configurator instance, to enable chaining of calls
+         */
+        public <T extends Model> SyncConfigurator mockFailure() {
```

Comment:
Can you change to accept a `DataStoreException`, and then pass in the failure when you make the call? That way, the `AppSyncMocking` can remain as utility software, with any specific values codified external to its implementation.

Reply:
Done